internal/pkg/otel/README.md

```diff
@@ -31,8 +31,8 @@ This section provides a summary of components included in the Elastic Distributi
 
 | Component | Version |
```

Comment:
@andrzej-stencel we run `mage update` from root dir to update this file. it gets regenerated based on versions loaded from `go.mod`